src/protocol/bolt/src/main/java/org/apache/jmeter/protocol/bolt/config/BoltConnectionElementBeanInfo.java

```diff
@@ -44,6 +44,9 @@ public BoltConnectionElementBeanInfo() {
         propertyDescriptor = property("password", TypeEditor.PasswordEditor);
         propertyDescriptor.setValue(NOT_UNDEFINED, Boolean.TRUE);
         propertyDescriptor.setValue(DEFAULT, "");
+        propertyDescriptor = property("database");
+        propertyDescriptor.setValue(NOT_UNDEFINED, Boolean.TRUE);
```

Comment:
This is going to break existing test plans. It should allow undefined database name